tests/inputFields.spec.ts

```diff
@@ -0,0 +1,95 @@
+import { test, expect } from '@playwright/test';
+
+test.beforeEach( async({page}) => {
+  await page.goto('/')
+  //1. Select the PET TYPES menu item in the navigation bar
+  await page.getByText('Pet Types').click()
+  //2. Add assertion of the "Pet Types" text displayed above the table with the list of pet types
+  expect(await page.locator('h2').getByText('Pet Types').isVisible()).toBeTruthy()
+})
+
+test('Update pet type', async ({page}) => {
+  test.slow()
+  //3. Click on "Edit" button for the "cat" pet type
+  //assumes cat is always first
+  await page.getByText('Edit').first().click()
+
+  //4. Add assertion of the "Edit Pet Type" text displayed
+  expect(await page.getByText('Edit Pet Type').isVisible()).toBeTruthy()
+
+  //5. Change the pet type name from "cat" to "rabbit" and click "Update" button
+  const textField = page.locator('#name')
+  await textField.click()
+  await textField.clear()
+  await textField.pressSequentially('rabbit')
+  await page.locator('button').getByText('Update').click()
+  
+  //6. Add the assertion that the first pet type in the list of types has a value "rabbit" 
+  test.slow()
+  const rabbit = page.locator('input[name=pettype_name]').first().inputValue()
+  expect(await rabbit).toBe('rabbit')
```

Comment:
Lines 29 and 30 can be shortened into a single line using a Locator Assertion.
Also I would recommend to read this blog post: https://www.bondaracademy.com/blog/tobe-vs-toequal-assertion-playwright

Modify all similar assertions later in your code to a single line locator assertion.